src/components/formItem/formItem.svelte

```diff
@@ -161,12 +161,12 @@
     justify-content: stretch;
     font: var(--font);
     gap: var(--gap);
-    transition:
-      color 0.2s ease-in-out;
+    transition: color 0.2s ease-in-out;
 
     .content {
+      flex: 1;
+      width: 100%;
```

Comment:
hey, just tested and I don't think we need this `width: 100%` - it comes from the flex: 1

Reply:
```suggestion
```